parametric/test_headers_b3multi.py

```diff
@@ -54,7 +53,7 @@ def test_headers_b3multi_extract_valid(test_agent, test_library):
 
 @enable_b3multi()
 @pytest.mark.skip_library("golang", "not implemented")
-@pytest.mark.skip_library("nodejs", "not implemented")
+@pytest.mark.skip_library("nodejs", "parent_id is 0 when extraction is invalid")
```

Comment:
As we discussed offline, since this is the current behavior we should be more lax and allow a parent_id of 0 in the system-tests, and we can enforce a single behavior later. To do that, we'll probably need to update the `x-b3-spanid` to a non-zero value and assert that the parent_id is 0 or null. Can you do that in this PR?